src/components/input/index.js

```diff
@@ -0,0 +1,15 @@
+import { useCallback } from 'react';
+import classes from './styles.module.css';
+
+const Input = props => {
+    const { inputTextVal, handleInputTextChange } = props;
+    const handleChange = useCallback(e => {
+        handleInputTextChange(e.target.value);
+    }, [inputTextVal]);
+
+    return <div className={classes.inputContainer}>
+        <textarea id='textArea' onChange={handleChange} placeholder="Enter Text" value={inputTextVal}/>
```

Comment:
remove id, and get placeholder from props too

Reply:
also all the components should have a style prop as well, to override default styles

Reply:
Done